tests/testthat/test-two-column-layouts.R

```diff
@@ -53,7 +53,7 @@ test_that("two_column_layout saving works", {
   my_output <- gt_two_column_layout(my_tables)
 
   path_2 <- tempfile(fileext = ".png")
-  on.exit(unlink(path_2))
+  on.exit(unlink(path_2), add = TRUE)
```

Comment:
This is likely one instance where you were leaving a temporary file behind on CRAN. If you don't have `add = TRUE`, the second `unlink()` call _clobbers_ the first, i.e. replaces it.

(This is an argument for preferring withr functions for this sort of work, but I won't go there just now. But, if using `unlink()`, the basic rule is to always specify `add = TRUE` unless you really do want to replace the previously scheduled cleanup.)

You might not need the `skip_on_cran()` here once we fix this.